src/Homework/Week3/Pb3/testPb3.java

```diff
@@ -0,0 +1,74 @@
+package Homework.Week3.Pb3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class testPb3 {
+
+    public static void main(String[] args) {
+        ArrayList<User> allUsers = new ArrayList<User>();
+        Board board = new Board();
+        Scanner s = new Scanner(System.in);
+        User currentUser = null;
+        while(true)
+        {
+            System.out.print("Enter a command and press enter: ");
```

Comment:
f bine ca ai implementat un scenariu mai complex pentru testarea boardului, ar trebui sa afisezi la consola si care sunt comenzile posibile pentru ca cel care testeaza sa aiba toate informatiile necesare, sa nu fie ascunse in cod; si pentru ca e un numar limitat de comenzi, ar merge sa fac un enum pentru comenzile astea